src/js/builtins/Module.ts

```diff
@@ -3,14 +3,16 @@ export function main() {
   return $requireMap.$get(Bun.main);
 }
 
-$visibility = "Private";
+$visibility = "PrivateRecursive";
+$alwaysInline;
 export function require(this: CommonJSModuleRecord, id: string) {
   return $tailCallForwardArguments($overridableRequire, this);
 }
 
 // overridableRequire can be overridden by setting `Module.prototype.require`
 $overriddenName = "require";
-$visibility = "Private";
+$visibility = "PrivateRecursive";
+$alwaysInline;
```

Comment:
similarly on this one, i wonder if this actually ever gets inlined, considering it is a part of the `tailCallForwardArguments`.

Reply:
My concern is that this would hide things we DON'T want hidden too.